rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,157 @@
+
+    ```{bash}
+    dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+    ```
+
+1. Enable the "Virtual Machine Platform" feature by running the following command in the PowerShell:
+
+    ```{bash}
+    dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+    ```
+
+1. Restart your system.
+1. Open "Windows PowerShell" with admin privileges again.
+1. Set WSL 2 as your default version using the following command in the PowerShell:
+
+    ```{bash}
+    wsl --set-default-version 2
+    ```
+
+1. Download and install [the latest WSL2 Linux kernel](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi) if you see the following message:
+
+    > WSL 2 requires an update to its kernel component. For information please visit `https://aka.ms/wsl2kernel`
+
+## Linux Distribution Setup {#linux-setup}
+
+After the [WSL 2 feature](#wsl-setup) has been enabled, you have to download and install a Linux distribution from [Microsoft Store](https://aka.ms/wslstore){target="_blank"}.
+
+Alternatively, you can also download and manually install Linux distros by clicking any of the links below:
+
+* [Ubuntu 20.04](https://aka.ms/wslubuntu2004)
+* [Ubuntu 20.04 ARM](https://aka.ms/wslubuntu2004arm)
+* [Ubuntu 18.04](https://aka.ms/wsl-ubuntu-1804)
+* [Ubuntu 18.04 ARM](https://aka.ms/wsl-ubuntu-1804-arm)
+* [Ubuntu 16.04](https://aka.ms/wsl-ubuntu-1604)
+* [Debian GNU/Linux](https://aka.ms/wsl-debian-gnulinux)
+* [Kali Linux](https://aka.ms/wsl-kali-linux-new)
+* [OpenSUSE Leap 42](https://aka.ms/wsl-opensuse-42)
+* [SUSE Linux Enterprise Server 12](https://aka.ms/wsl-sles-12)
+* [Fedora Remix for WSL](https://github.com/WhitewaterFoundry/WSLFedoraRemix/releases/)
+
+We will download and install ["Ubuntu 20.04"](https://aka.ms/wslubuntu2004) as an example for this article.
+
+Once the distribution has been installed and launched, you will be prompted the initial user configuration like below:
+
+> Installing, this may take a few minutes...                                                                                                             
+> 
+> Please create a default UNIX user account. The username does not need to match your Windows username.                                                  
+> For more information visit: `https://aka.ms/wslusers` 
+> 
+> Enter new UNIX username:                                                                                                                               
+  
+Provide a user name followed by password that will be used for both your WSL distribution and RStudio Server authentication later on.
+
+You can now close the WSL instance by typing the following command:
+   
+```{bash}
+exit
+```
+
+* **Note:** Unlike the PowerShell, you cannot use `Ctrl + V` shortcut key to paste the clipboard content into the WSL terminal. Alternatively, press mouse right-click button to emulate the paste function.
```

Comment:
Actually, the "Command Prompt" in Windows-10 2004 does have an option to enable using Ctrl + Shift + V for pasting and Ctrl + Shift + C for copy! It's in the Command Prompt Properties under Edit Options. Up to you if you want to mention this but it is very handy.